.travis.yml

```diff
@@ -28,59 +28,57 @@ matrix:
     - python: 3.8
       env: TOXENV=mypy
 
-    - python: 2.7
-      env: TOXENV=coverage-py27-tw171,codecov
     - python: 2.7
       env: TOXENV=coverage-py27-tw184,codecov
+    - python: 2.7
+      env: TOXENV=coverage-py27-tw192,codecov
     - python: 2.7
       env: TOXENV=coverage-py27-twcurrent,codecov
```

Comment:
Should this be `coverage-py27-tw203` so that it doesn't start failing the next time a Twisted release occurs?

Reply:
`twcurrent` tries to avoid that. We should add `tw203` at some point this year, but I usually wait a bit further past the first release the year so that `twcurrent` is less redundant and keep the build times manageable.